Cargo.toml

```diff
@@ -29,86 +29,170 @@ license = "BSD-3-Clause-Clear"
+#
+# MAINTAIN ALPHABETICAL ORDER for all dependencies within each section
 
-alloy-dyn-abi = "=1.3.1"
-alloy-primitives = "=1.3.1"
-alloy-sol-types = "=1.3.1"
-alloy-signer = "=1.0.30"
-alloy-signer-local = "=1.0.30"
+# Internal Zama modules
+backward-compatibility = { path = "./backward-compatibility" }  # Internal backward compatibility testing
+bc2wrap = { path = "./bc2wrap" }  # Internal bincode wrapper - MEDIUM RISK: Internal custom implementation
+kms = { path = "./core/service", default-features = false }  # Core KMS service - LOW RISK: Internal Zama core module
+kms-grpc = { path = "./core/grpc", default-features = false }  # KMS gRPC interface - LOW RISK: Internal Zama module
+observability = { path = "./observability" }  # Internal observability module - LOW RISK: Internal Zama module
+threshold-fhe = { path = "./core/threshold/", default-features = false }  # Threshold FHE implementation - LOW RISK: Internal Zama core module
+
+# External dependencies (alphabetically sorted)
+aes = "=0.8.4"  # AES encryption - LOW RISK: RustCrypto org, very popular (40M+ downloads), actively maintained
+aes-gcm = { version = "=0.10.3", features = ["std"] }  # AES-GCM authenticated encryption - LOW RISK: RustCrypto org, 10M+ downloads
+aes-gcm-siv = "=0.11.1"  # AES-GCM-SIV authenticated encryption - LOW RISK: RustCrypto org, nonce misuse resistant, 500K+ downloads
+aes-prng = "=0.2.1"  # AES-based PRNG for deterministic randomness - LOW RISK: Maintained by @mortendahl (ZAMA) and Dragos
+alloy-dyn-abi = "=1.3.1"  # Ethereum ABI encoding - LOW RISK: Alloy/Paradigm team, reputable, growing ecosystem
+alloy-primitives = "=1.3.1"  # Ethereum primitives - LOW RISK: Alloy/Paradigm team
+alloy-signer = "=1.0.30"  # Ethereum signing abstraction - LOW RISK: Alloy/Paradigm team
+alloy-signer-local = "=1.0.30"  # Local Ethereum signer - LOW RISK: Alloy/Paradigm team
+alloy-sol-types = "=1.3.1"  # Solidity types & EIP-712 - LOW RISK: Alloy/Paradigm team
+anyhow = "=1.0.98"  # Error handling - LOW RISK: dtolnay (very reputable), 100M+ downloads
+assert_cmd = "=2.0.17"  # Assert command for testing - LOW RISK: assert-rs team, 1M+ downloads
+async-std = { version = "=1.13.1", features = ["attributes", "tokio1"] }  # Async runtime for testing - LOW RISK: async-rs team, 10M+ downloads
+async-trait = "=0.1.88"  # Async trait support - LOW RISK: dtolnay, widely used
+async_cell = "0.2.2"  # Async cell implementation - MEDIUM RISK: Individual maintainer, very low popularity, consider alternatives
+attestation-doc-validation = { version = "=0.10.0" }  # AWS Nitro attestation validation - LOW RISK: Evervault (reputable security company), security-critical but trusted
+aws-config = { version = "=1.5.9" }  # AWS SDK configuration - LOW RISK: Official AWS SDK, actively maintained
+aws-nitro-enclaves-nsm-api = { version = "=0.4.0" }  # AWS Nitro Enclaves NSM API - MEDIUM RISK: Security-critical enclave operations, needs audit
```

Comment:
Isn't this owned by AWS? Should the risk not be considered low then?

Reply:
here was meant a constant MEDIUM status due to importance of the crate. I.e. every version change shall be considered carefully + if in future we'll have an audit process - than medium risk deps will be always treated separately with high attention. But agree, maybe let's treat it LOW due to AWS maintenance

Reply:
Addressed in https://github.com/zama-ai/kms/pull/132/commits/7eeb41dd26969d7b52a4654e15c6ea7ee0530621